dashboard/app/models/teacher_feedback.rb

```diff
@@ -150,6 +154,51 @@ def summarize(is_latest = false)
     }
   end
 
+  # Passing level, script_level and student into summarize_for_csv since it's
+  # called in a loop to avoid N+1s
+  def summarize_for_csv(level, script_level, student, sublevel_index = nil)
+    rubric_performance_json_to_ruby = {
+      performanceLevel1: "rubric_performance_level_1",
+      performanceLevel2: "rubric_performance_level_2",
+      performanceLevel3: "rubric_performance_level_3",
+      performanceLevel4: "rubric_performance_level_4"
+    }
+
+    rubric_performance_headers = {
+      performanceLevel1: "Extensive Evidence",
+      performanceLevel2: "Convincing Evidence",
+      performanceLevel3: "Limited Evidence",
+      performanceLevel4: "No Evidence"
+    }
+
+    review_state_labels = {
+      keepWorking: "Needs more work",
+      completed: "Reviewed, completed",
+    }
+
+    review_state_label = awaiting_teacher_review?(true) ? "Waiting for review" : review_state_labels[review_state&.to_sym]
+    level_num = script_level.position.to_s
+
+    if sublevel_index
+      alphabet = ('a'..'z').to_a
```

Comment:
Creating this array every time seems a bit wasteful.  Maybe save the alphabet in a class variable or use `ord` and `chr` instead?

Reply:
I've moved it to BubbleChoice::ALPHABET, because it's also used there, seem reasonable?